maps/tether/tether_areas.dm

```diff
@@ -221,6 +221,12 @@
 	name = "\improper Bar Backroom"
 	icon_state = "red"
 	sound_env = SMALL_SOFTFLOOR
+/area/tether/surfacebase/servicebackroom
+	name = "\improper Service Block Backroom"
+	icon_state = "red"
+/area/tether/surfacebase/maintdice
+	name = "\improper Bar Back Maint"
```

Comment:
Imo, official area names should be full, so "Maintenance"

Reply:
That's reasonable.

Reply:
This is fixed and changed to "Bar Back Maintenance" and "/barbackmaintenance".